test/CompressionPlugin.test.js

```diff
@@ -0,0 +1,32 @@
+import Plugin from '../src/index';
+import { cleanErrorStack, createCompiler, compile } from './helpers';
+
+describe('CompressionPlugin', () => {
+  let compiler;
+
+  beforeEach(() => {
+    compiler = createCompiler({
+      entry: {
+        js: `${__dirname}/fixtures/entry.js`,
+      },
+      output: {
+        path: `${__dirname}/dist`,
+        filename: '[name].js?var=[hash]',
+        chunkFilename: '[id].[name].js?ver=[hash]',
+      },
+    });
+  });
+
+  it('should works (without options)', () => {
+    new Plugin().apply(compiler);
```

Comment:
```js
const config = {
   plugins: [
      new Plugin(options)
   ]
}
```

Reply:
@michael-ciniawsky no, it is impossible test some features like errors inside `stats`